functions/tsconfig.json

```diff
@@ -3,6 +3,8 @@
     "module": "esnext",
     "moduleResolution": "node",
     "allowSyntheticDefaultImports": true,
+    "allowJs": true,
```

Comment:
Are we using plain JS in src folder? I thought everything was .ts?

Reply:
good catch, i removed this line